examples/context-based-restrictions.v1.test.js

```diff
@@ -0,0 +1,528 @@
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+
+//
+// This file provides an example of how to use the Context Based Restrictions service.
+//
+// The following configuration properties are assumed to be defined:
+// CONTEXT_BASED_RESTRICTIONS_URL=<service base url>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_TYPE=iam
+// CONTEXT_BASED_RESTRICTIONS_APIKEY=<IAM apikey>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+// CONTEXT_BASED_RESTRICTIONS_TEST_ACCOUNT_ID=<the id of the account under which test CBR zones and rules are created>
+// CONTEXT_BASED_RESTRICTIONS_TEST_SERVICE_NAME=<the name of the service to be associated with the test CBR rules>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+const configFile = 'context_based_restrictions_v1.env';
+
+const describe = authHelper.prepareTests(configFile);
+
+// Save original console.log
+const originalLog = console.log;
+
+// Mocks for console.log and console.warn
+const consoleLogMock = jest.spyOn(console, 'log');
+const consoleWarnMock = jest.spyOn(console, 'warn');
+
+describe('ContextBasedRestrictionsV1', () => {
+  // begin-common
+
+  const contextBasedRestrictionsService = ContextBasedRestrictionsV1.newInstance({});
+
+  // end-common
+
+  // Access additional configuration values from external file
+  const config = readExternalSources(ContextBasedRestrictionsV1.DEFAULT_SERVICE_NAME);
+  const {
+    apikey: apiKey,
+    url: URL,
+    authUrl,
+    authType,
+    testAccountId: accountId,
+    testServiceName: serviceName,
+  } = config;
+
+  expect(config).not.toBeNull();
+  expect(accountId).not.toBeNull();
+  expect(accountId).toBeDefined();
+  expect(apiKey).not.toBeNull();
+  expect(apiKey).toBeDefined();
+  expect(URL).not.toBeNull();
+  expect(URL).toBeDefined();
+  expect(authUrl).not.toBeNull();
+  expect(authUrl).toBeDefined();
+  expect(authType).not.toBeNull();
+  expect(authType).toBeDefined();
+  expect(serviceName).not.toBeNull();
+  expect(serviceName).toBeDefined();
+
+  let zoneId;
+  let zoneEtag;
+  let ruleId;
+  let ruleEtag;
+
+  test('createZone request example', async (done) => {
+    consoleLogMock.mockImplementation((output) => {
+      originalLog(output);
+      done();
+    });
+    consoleWarnMock.mockImplementation((output) => {
+      done(output);
+    });
+
+    originalLog('createZone() result:');
+    // begin-create_zone
+
+    // Request models needed by this operation.
+
+    // AddressIPAddress
+    const addressModel = {
+      type: 'ipAddress',
+      value: '169.23.56.234',
+    };
+
+    const params = {
+      name: 'an example of zone',
+      accountId,
+      addresses: [addressModel],
+      description: 'this is an example of zone',
+    };
```

Comment:
I'd like to ask you to match the `name` and `description` property values to what the 3 other SDKs have.

Reply:
checked, they are same as other SDKs. no need to change